core/server/common/src/main/java/alluxio/master/Master.java

```diff
@@ -24,7 +22,7 @@
 /**
  * This interface contains common operations for all masters.
  */
-public interface Master extends JournalCheckpointStreamable {
+public interface Master extends Iterable<JournalEntry> {
```

Comment:
I think we should have a separate method in the Master interface instead of reusing this interface since the `Master` is not a collection and we would not run `foreach` on a Master.